CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.154.3, 19 June 2021
+
+- fix: Fetches upload-pack using git if http fails
```

Comment:
Question: some of these are community contributions, do those contributors want a shout-out?

I tend to include a `(@awesome-contributor)` for anyone outside GitHub's Dependabot team as means of thanks and encouragement.
It's one of those weird things that's done but not written down anywhere. We could not care, we could write it down, or encode it in the [bump-version](https://github.com/dependabot/dependabot-core/blob/main/bin/bump-version.rb#L37) script

Reply:
That's a great idea.